### PR TITLE
ACP2E-339: [Documentation] Add clarification that automatic sorting is applied to the category by the "Save Category" button only

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

In scope of MC-42418 ticket investigation it was found that current clarification, that category should be re-saved, doesn't include automatic sorting words.
"Automatic sorting" wording should be included in the rule to avoid Merchants confusion in the future and to clarify this rule more clear.
Also this tip should be moved to the upper part of the document to be visible to the Merchants.

Document to update:
https://docs.magento.com/user-guide/catalog/categories-category-products.html

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

Magento 2.4.x

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

Adobe Commerce only

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

No

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/categories-category-products.html

### CODE REVIEW

File: src/catalog/categories-category-products.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ title: Products in Category
 
 The Products in Category section lists the products that are currently assigned to the category. The search filters at the top of each column are used to add and remove products from the category. You can also use [category rules]({% link catalog/category-product-rules.md %}) (<span class="ee-only"></span>) to dynamically change the product selection when a set of conditions is met. To learn more, see [Visual Merchandiser]({% link marketing/visual-merchandiser.md %}).
 
+{:.bs-callout-tip}
+When setting up a category rule, the products are _sorted_, _matched_, _assigned_ and _unassigned_ according to this rule when this category is saved **only**. For example, if you add a new product to the catalog and want to assign it according to the rule, you must **re-save each category** that is set to match products by rule to ensure that the new product is assigned. And if any product stock status is changed to `In Stock` or `Out of Stock`, and products in category should be _sorted_ according to the **Automatic Sorting** rule, you must click <span class="btn">Save Category</span> as well.
```
Comment (leslietilling) [inline src/catalog/categories-category-products.md:8]:
```suggestion
During category rule setup, the products are _sorted_, _matched_, _assigned_, and _unassigned_ according to that rule **_only_** when this category is saved. For example, if you add a new product to the catalog and want to assign it according to the rule, you **must re-save each category** that is set to match products by rule to ensure that the new product is assigned. Also, if any product stock status is changed to `In Stock` or `Out of Stock` and products in the category should be _sorted_ according to the **Automatic Sorting** rule, you must click <span class="btn">Save Category</span> as well.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just minor tweaks for clarity

File: src/catalog/categories-category-products.md (hunk 1 above)
Comment (vkolesny) [reply src/catalog/categories-category-products.md]: Suggested changes were committed.
Ready for review.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**